src/components/ToolBar/ColorButton.vue

```diff
@@ -1,5 +1,5 @@
 <script setup lang="ts">
-import { type Color } from '@/components/ToolBar.vue'
+import { type Color } from '@/components/SubToolMenu.vue'
 
 interface Props {
```

Comment:
```suggestion
interface Props {
  color: Color
  index: number
  activeIndex: number | null
}
```
activeindex -> activeIndex
コーディング規約決めてないですが...
変数名(2単語以上)なので、キャメルケースで記入を

Reply:
ありがとうございます。修正します！

Reply:
Linterで制御できた気がするのでやってみます